docs/source/fundamentals.md

```diff
@@ -101,29 +99,155 @@ Once a certain degree of interoperability has been reached, other interesting co
   is one of the principles of SimPhoNy. For that, ontologies play a major role.
 
-#### Requirement simplification
-  Since we know what a user means from the semantic approach, we can use this to automatise and simplify
-  the setup and initialisation of processes using default settings.
+### Ontology
+```eval_rst
+.. important::
+   An ontology is a formal specification of a shared conceptualization.  `[Borst, 1997]
+   <https://research.utwente.nl/en/publications/construction-of-engineering-ontologies-for-knowledge-sharing-and->`_ .
+
+```
+
+Let's look at the individual components of this definition, starting from the end.
+ - _Conceptualization_, an ontology will work on the ideas and relationships in an area of interest.
+ - _Shared_, the ideas and concepts are perceived and agreed by multiple people.
+ - _Specification_, it will define and describe them in detail, following some predetermined rules and format.
+ - _Formal_, meaning it will follow a machine readable syntax.
+
+In a simpler way, an ontology can be seen as the definition of concepts relevant to a given domain, 
+as well as the relationships between them, in a way that a machine can interpret it.
+
+For a deeper, more detailed analysis of the definition, refer to [[Guarino, 2009]](http://dx.doi.org/10.1007/978-3-540-92673-3_0).
+
+Ontologies are more elaborated than taxonomies in that they can include multiple kinds of relationships
+(not just parent-child) between complex concepts in big domains.
+
+#### EMMO
+TODO
+### CUDS
+Cuds, or Common Universal Data Structure, is the ontology compliant data format of OSP-core:
+- **Cuds is an ontology individual**: each Cuds object is an instantiation of a class in the ontology.
+  If there is a food ontology, describing classes like pizza or pasta, a Cuds object could represent one specific pizza or pasta dish, that exists in the real world.
+  Like ontology individuals, Cuds objects can be related with other individuals/Cuds by relations defined in the ontology. Like a _pizza_ that 'hasPart' _tomato sauce_
+- **Cuds is API**: To allow users to interact with the ontology individuals and their data, Cuds provide a CRUD API.
+- **Cuds is a container**: Depending on the relationship connecting two Cuds objects, a certain instance can be seen as a container of other instances.
+  We call a relationship that express containment an 'active relationship'.
+  In the pizza example, 'hasPart' would be an 'active relationship'. If one would like to share the pizza Cuds object with others, one would like to share also the tomato sauce.
+- **Cuds is RDF**: Internally a Cuds object is only an interface to an RDF-based triple store that contains the data of all Cuds objects.
+- **Cuds is a node in a graph**: : Cuds being individuals in an RDF graph implies that each Cuds object can also be seen as a node in a graph.
+  This does not conflict with the container perspective, instead we see it as to different views on the data.
```

Comment:
For me, a CUDS object is a Python representation of an ontology individual that has an interactive interface. But that's not to say that the other definitions are not correct.